### PR TITLE
Adopt oc scale module for openshift_logging role

### PR DESCRIPTION
**Pull request by jcantrill**
cc @kwoodson  trying to use 'oc_scale' module encountered:

```
TASK [openshift_logging : stop elasticsearch] **********************************
task path: /home/jeff.cantrill/git/openshift-ansible/roles/openshift_logging/tasks/stop_cluster.yaml:23
failed: [openshift] (item=deploymentconfig/logging-es-3zpofafu) => {"failed": true, "module_stderr": "Shared connection to 192.168.121.194 closed.\r\n", "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_M87iwH/ansible_module_oc_scale.py\", line 1707, in <module>\r\n    main()\r\n  File \"/tmp/ansible_M87iwH/ansible_module_oc_scale.py\", line 1699, in main\r\n    rval = OCScale.run_ansible(module.params, module.check_mode)\r\n  File \"/tmp/ansible_M87iwH/ansible_module_oc_scale.py\", line 1658, in run_ansible\r\n    if oc_scale.needs_update():\r\n  File \"/tmp/ansible_M87iwH/ansible_module_oc_scale.py\", line 1630, in needs_update\r\n    return self.resource.needs_update_replicas(self.replicas)\r\nAttributeError: 'NoneType' object has no attribute 'needs_update_replicas'\r\n", "msg": "MODULE FAILURE", "object": "deploymentconfig/logging-es-3zpofafu"}
	to retry, use: --limit @/home/jeff.cantrill/git/openshift-ansible/playbooks/common/openshift-cluster/openshift_logging.retry

```

### CODE REVIEW

**Comment by kwoodson** [comment]
@jcantrill 

Thanks for finding this.  This happens when the object you are attempting to scale is non-existent.  I have added the code to fail the module call when that happens.  I also added test cases.  Here is the PR. https://github.com/openshift/openshift-ansible/pull/3206

File: roles/openshift_logging/tasks/start_cluster.yaml [hunk 1]
```diff
@@ -23,9 +23,11 @@
   changed_when: no
 
 - name: start elasticsearch
-  include: scale.yaml
-  vars:
-    desired: 1
+  oc_scale:
+    kind: dc
+    name: "{{object.split('/')[1]}}"
+    namespace: "{{mktemp.stdout}}/admin.kubeconfig"
```
Comment (kwoodson) [inline roles/openshift_logging/tasks/start_cluster.yaml]: Looks like these params were swapped.  Probably copy pasta.

`namespace: "{{openshift_logging_namespace}}"`
`kubeconfig: "{{mktemp.stdout}}/admin.kubeconfig"`

File: roles/openshift_logging/tasks/start_cluster.yaml [hunk 2]
```diff
@@ -37,9 +39,11 @@
   changed_when: no
 
 - name: start kibana
-  include: scale.yaml
-  vars:
-    desired: "{{ openshift_logging_kibana_replica_count | default (1) }}"
+  oc_scale:
+    kind: dc
+    name: "{{object.split('/')[1]}}"
+    namespace: "{{mktemp.stdout}}/admin.kubeconfig"
```
Comment (kwoodson) [inline roles/openshift_logging/tasks/start_cluster.yaml]: same as above.  I also noticed others here.  Probably a search and replace should fix this.

File: roles/openshift_logging/tasks/start_cluster.yaml (hunk 1 above)
Comment (jcantrill) [inline roles/openshift_logging/tasks/start_cluster.yaml]: fixed.  Thanks for catching... it was late.

File: roles/openshift_logging/tasks/start_cluster.yaml (hunk 2 above)
Comment (jcantrill) [inline roles/openshift_logging/tasks/start_cluster.yaml]: fixed.  Thanks for catching... it was late.

**Comment by ewolinetz** [comment]
aos-ci-test

**Pull request merged**